apps/allocations/contracts/Allocations.sol

```diff
@@ -174,13 +174,12 @@ contract Allocations is AragonApp, Fundable {
 
     function runPayout(uint256 _payoutId) external payable isInitialized auth(EXECUTE_PAYOUT_ROLE) returns(bool success) {
         Payout storage payout = payouts[_payoutId];
-        uint256 pointsPer;
         uint256 totalSupport;
         uint i;
         for (i = 0; i < payout.supports.length; i++) {
             totalSupport += payout.supports[i];
         }
-
+        totalSupport = totalSupport.div(10000);
```

Comment:
We should not use random operations with literal values, better use constants and define the behavior on this operation

Reply:
Outdated